### ISSUE TITLE
Sign in with Google temporarily disabled for this app

### ISSUE THREAD

**Issue by bakoeldesain** [andresoviedo/google-drive-ftp-adapter#46]
when I tried to execute google-drive-ftp-adapter, I always stuck at this message : "google-drive-ftp-adapter", please advise.

**Comment by enriiquee** [comment]
I have the same issue. In my case when I tried to log in my account, google shows this message: "se ha inhabilitado temporalmente el inicio de sesion con google en esta aplicaicon".

**Comment by HPerrett** [comment]
+1 to this issue

**Comment by gongtao0607** [comment]
Same issue here

**Comment by koleon03** [comment]
Also having this issue

**Comment by seiljang0220** [comment]
I have the same problem, too.

**Comment by regularperson2586** [comment]
same issue :/

**Comment by andresoviedo** [comment]
Hello. I've sent the Google Drive API verification request for the app to Google Cloud People.
It says 1-2 weeks to respond.
If verification is accepted, then you will be able to use the app again.
Regards

**Comment by krumaska** [comment]
![image](https://user-images.githubusercontent.com/51515055/72422128-31b12780-37c5-11ea-9f39-86113fb8d3ab.png)  
I use a window server 2003 and it works well.  
But I'm using Windows 10 in pc. It still not working.

**Comment by Ufinexa** [comment]
Is there any progress on the Google Drive API verification request? I am getting the same issue as everyone else still

**Comment by Youboontoo** [comment]
Still not working here. Not too sure what the hangup is if the verification request was sent over a month ago.

**Comment by OfficialNova** [comment]
Not working here either

**Comment by useribs** [comment]
Still the same issue, which makes this app unusable! Plz help

**Comment by lexoyo** [comment]
You should build it with your own app key?

**Comment by mariolatiffathy** [comment]
**Here is a solution:**

1. Follow https://github.com/mbrother2/backuptogoogle/wiki/Create-own-Google-credential-step-by-step to get your own OAuth credentials.

2. Install `Java 8`, `Maven 3`, and `git` on your server.

3. Clone the project by:
```
git clone https://github.com/andresoviedo/google-drive-ftp-adapter.git
cd google-drive-ftp-adapter
```

4. Edit the `src/main/resources/client_secrets.json` file and put the OAuth client ID and client secret that you got.

5. Compile the project by:
```
mvn clean package
```

**Comment by rltim151** [comment]
> **Here is a solution:**
> 
> 1. Follow https://github.com/mbrother2/backuptogoogle/wiki/Create-own-Google-credential-step-by-step to get your own OAuth credentials.
> 2. Install `Java 8 JDK (not JRE)`, `Apache Maven 3`, and `git` on your server.
> 3. Clone the project by:
> 
> ```
> git clone https://github.com/andresoviedo/google-drive-ftp-adapter.git
> cd google-drive-ftp-adapter
> ```
> 
> 1. Edit the `src/main/resources/client_secrets.json` file and put the OAuth client ID and client secret that you got from step 1.
> 2. Compile the project by:
> 
> ```
> mvn clean package
> ```
> 
> 1. The JAR file (named `google-drive-ftp-adapter-1.6.2-jar-with-dependencies.jar`) will be available in a directory named `target`
> 
> **EDIT:** There's a known error when you execute `mvn clean package`, that error is `Perhaps you are running on a JRE rather than a JDK?`... This happens because you have the Java 8 JRE only and not the JDK. See https://stackoverflow.com/a/40642204 for a Linux solution.
> 
> You can also confirm if you have the JDK or no by typing `javac -version`

is the error on linux or windows?

**Comment by rltim151** [comment]
awesome it worked! Thanks alot!

**Comment by mariolatiffathy** [comment]
> > **Here is a solution:**
> > 
> > 1. Follow https://github.com/mbrother2/backuptogoogle/wiki/Create-own-Google-credential-step-by-step to get your own OAuth credentials.
> > 2. Install `Java 8 JDK (not JRE)`, `Apache Maven 3`, and `git` on your server.
> > 3. Clone the project by:
> > 
> > ```
> > git clone https://github.com/andresoviedo/google-drive-ftp-adapter.git
> > cd google-drive-ftp-adapter
> > ```
> > 
> > 
> > 
> > 1. Edit the `src/main/resources/client_secrets.json` file and put the OAuth client ID and client secret that you got from step 1.
> > 2. Compile the project by:
> > 
> > ```
> > mvn clean package
> > ```
> > 
> > 
> > 
> > 1. The JAR file (named `google-drive-ftp-adapter-1.6.2-jar-with-dependencies.jar`) will be available in a directory named `target`
> > 
> > **EDIT:** There's a known error when you execute `mvn clean package`, that error is `Perhaps you are running on a JRE rather than a JDK?`... This happens because you have the Java 8 JRE only and not the JDK. See https://stackoverflow.com/a/40642204 for a Linux solution.
> > You can also confirm if you have the JDK or no by typing `javac -version`
> 
> is the error on linux or windows?

The error will appear as long as Java JDK isn't installed.

**Comment by joaofgoncalves** [comment]
Yep! Also worked here (on Windows 10). Great fix. If you are using this OS don't forget to set JAVA_HOME env variable to the installed JDK 1.8 directory otherwise `mvn clean package` won't work properly.
This workaround will throw a similar error first but then allows to create an exception for the app set in step 1 of @mariolatiffathy solution. After that connecting via FTP to GDrive in FileZilla is straightforward. 
Cheers!

**Comment by francoross** [comment]
I get OAuth credentials after followed **oddmario** tutorial but I use Windows 10. 
Where do I insert these credentials? 
Do I need to install Java 8 SDK?

![access denied](https://user-images.githubusercontent.com/76166342/114545293-13eaa080-9c5c-11eb-9798-e1c876631fb6.png)

**Comment by francoross** [comment]
> Yep! Also worked here (on Windows 10). Great fix. If you are using this OS don't forget to set JAVA_HOME env variable to the installed JDK 1.8 directory otherwise `mvn clean package` won't work properly.
> This workaround will throw a similar error first but then allows to create an exception for the app set in step 1 of @mariolatiffathy solution. After that connecting via FTP to GDrive in FileZilla is straightforward.
> Cheers!

I change to _test_ into _devepment_ mode to avoid google veriication to start app but it tell me about a received verification code but I don't see any windows opened, where is google drive connector? In cmd I see this 

![updating app started ma dove si trova](https://user-images.githubusercontent.com/76166342/114547752-1995b580-9c5f-11eb-9c09-3989ff813cf3.png)

![received verification code](https://user-images.githubusercontent.com/76166342/114547636-f79c3300-9c5e-11eb-805b-99bf624bcdb1.png)

**Comment by francoross** [comment]
Ok, I solved, I read README and I set this credentials on Filezilla (host, username, password, port)

_localhost
user
user
1821_

![risolto con google ftp](https://user-images.githubusercontent.com/76166342/114548984-c3c20d00-9c60-11eb-814a-03034877cf5b.png)

Thank you for patience !

**Comment by philsward** [comment]
I may have very well glossed over the details, BUT... For anyone struggling with the Pathing on Windows because you're like me and skim through everything then play around until you figure it out instead of digesting it, you probably need to have the path to both maven and java. (see the last entries of the image) This IN ADDITION to the JAVA_HOME

![image](https://user-images.githubusercontent.com/3436110/167698136-bbda6453-f5a8-4fe0-b7ef-6d676bd88978.png)

So:

JAVA_HOME = C:\Program Files\Java\jdk1.8.0_331\
PATH = C:\Program Files\Java\jdk1.8.0_331\bin;C:\Program Files\apache-maven-3.8.5\bin

A reboot is probably in order after adding these.

Once that is all done, open cmd and run `mvn clean package` inside the `.\google-drive-ftp-adapter-master` folder
